src/Symfony/Component/HttpFoundation/InputBag.php

```diff
@@ -90,10 +90,10 @@ public function set(string $key, mixed $value): void
      *
      * @throws BadRequestException if the input cannot be converted to an enum
      */
-    public function getEnum(string $key, string $class, ?\BackedEnum $default = null): ?\BackedEnum
+    public function getEnum(string $key, string $class, ?\BackedEnum $default = null, bool $defaultIfUnexpectedValue = false): ?\BackedEnum
```

Comment:
Adding a parameter to a non-final method is not allowed by [our BC policy](https://symfony.com/doc/current/contributing/code/bc.html#changing-classes).

We have a workaround using `func_get_arg` and the `@param` annotation.